src/main/java/uk/gov/hmcts/reform/idam/client/IdamApi.java

```diff
@@ -26,6 +28,17 @@ GeneratePinResponse generatePin(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
     );
 
+    @RequestMapping(
+        method = RequestMethod.GET,
+        value = "/pin",
+        consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE
+    )
+    Response authenticatePinUser(
```

Comment:
have you tested that this works?

Reply:
This was copied exactly from a working Divorce service so yes it's definitely working.